ITEM.tsv

```diff
@@ -74,7 +74,7 @@ ITEM_20150317_000073	Mandrapick
 ITEM_20150317_000074	This sword is said to clear its wielder's mind with its Mandragora essence. 
 ITEM_20150317_000075	Wizard Blade 
 ITEM_20150317_000076	You should never underestimate a sword wielding wizard, especially if that sword is the Wizard Blade. 
-ITEM_20150317_000077	Cheminis Sword 
+ITEM_20150317_000077	Chemini's Sword 
```

Comment:
https://github.com/Treeofsavior/EnglishTranslation/wiki/2.3.%20Equipment 

The original was fine.